src/sentry/sentry_metrics/configuration.py

```diff
@@ -47,4 +55,30 @@ def get_ingest_config(use_case_key: UseCaseKey) -> MetricsIngestConfiguration:
             )
         )
 
-    return _METRICS_INGEST_CONFIG_BY_USE_CASE[use_case_key]
+        # Make sure that if we run backends in parallel we aren't
+        # double producing to the snuba-metrics-generics topic
+        dummy_output_topic = settings.KAFKA_SNUBA_GENERICS_METRICS_DUMMY
```

Comment:
I'd call this _CLOUDSPANNER instead of _DUMMY. What's the reason for calling it dummy? Are you saying we will not be using it permanently after the migration?

Reply:
Well I was thinking we could use the same output topic for testing multiple backends, so if we wanted to try to test something out after cloudspanner it could work for that too.

Yeah in my mind this topic would just be used for testing

Reply:
Right it's a test now. But if we go ahead with this, won't it become the permanent topic when we cut over? IMO even for a test _CLOUDSPANNER is far more meaningful than dummy since and hardcoded here in the cloudspanner configuration.

Reply:
nit: There does not seem to be a lot of value in assigning `settings.KAFKA_SNUBA_GENERICS_METRICS_DUMMY` to a variable and then using that variable when you instantiate the Config object. It actually makes the code harder to read.

Reply:
Actually, I think there is a benefit to defining in settings. It's because the cluster mapping must be defined there. So it keeps those two things together.

Reply:
Yeah Im going to keep in settings, but it doesn't need to be assigned to a variable.

Reply:
It needs to be a variable if we want to import it here. Unless you are suggesting just duplicating the value?